### PR TITLE
[PREVIEW] RDM-2493: Date/Time should be displayed in local time zone

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-2493

### Change description ###
The event created date/time is saved in UTC however it  also displayed in UTC. This should instead be displayed in BST.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/shared/palette/utils/date.pipe.spec.ts [hunk 1]
```diff
@@ -3,99 +3,105 @@ import { DatePipe } from './date.pipe';
 describe('DatePipe', () => {
 
   let datePipe: DatePipe;
-  // const EXPECTED_OFFSET = - new Date().getTimezoneOffset() / 60;
-  const EXPECTED_OFFSET = 0; // RDM-1149 changed the pipe logic
+  const EXPECTED_OFFSET = - new Date().getTimezoneOffset() / 60;
+  // const EXPECTED_OFFSET = 0; // RDM-1149 changed the pipe logic
```
Comment (adr1ancho) [inline src/app/shared/palette/utils/date.pipe.spec.ts]: Line 7 is not required

File: src/app/shared/palette/utils/date.pipe.ts [hunk 2]
```diff
@@ -12,13 +12,19 @@ export class DatePipe implements PipeTransform {
     ['Jan'], ['Feb'], ['Mar'], ['Apr'], ['May'], ['Jun'], ['Jul'], ['Aug'], ['Sep'], ['Oct'], ['Nov'], ['Dec'],
   ];
 
-  transform(value: string, format: string): string {
+  transform(value: string, zone: string, format: string): string {
```
Comment (adr1ancho) [inline src/app/shared/palette/utils/date.pipe.ts]: Can we have one more function

transform(value: string, format: string): string {
    return transform(value, format, null)
}

Comment (MKaramath) [reply src/app/shared/palette/utils/date.pipe.ts]: As discussed, the method overloading is not supporting in angular.

File: src/app/shared/palette/utils/date.pipe.spec.ts (hunk 1 above)
Comment (MKaramath) [reply src/app/shared/palette/utils/date.pipe.spec.ts]: Done.

File: src/app/shared/palette/utils/date.pipe.ts (hunk 2 above)
Comment (adr1ancho) [reply src/app/shared/palette/utils/date.pipe.ts]: thx

**Pull request merged**